packages/labs/src/components/tag-input/tagInput.tsx

```diff
@@ -23,6 +23,13 @@ import {
 import * as Classes from "../../common/classes";
 
 export interface ITagInputProps extends IProps {
+    /**
+     * Whether the component is non-interactive.
+     * Note that you'll also need to disable the component's rightElement, if appropriate.
```

Comment:
nit: wrap comments to 80-char lines.

Reply:
nit: wrap in back ticks: `rightElement`

Reply:
@cmslewis does the linter not enforce comments < 80-char lines?

Reply:
Nah, it just enforces a global max of 120 chars. To achieve 80-char lines in comments, I use the VS Code `Rewrap` plugin: https://marketplace.visualstudio.com/items?itemName=stkb.rewrap